pages/dkp/konvoy/2.2/dkp-upgrade/upgrade-konvoy/essential/index.md

```diff
@@ -101,6 +110,21 @@ configmap/node-feature-discovery-my-aws-cluster upgraded
 clusterresourceset.addons.cluster.x-k8s.io/nvidia-feature-discovery-my-aws-cluster upgraded
 configmap/nvidia-feature-discovery-my-aws-cluster upgraded
 ```
+Flags:
+    -h, --help  Help for addons
+
+Global Flags:
+    -v, --verbose int   Output verbostiy
+
+For more information about a command, use:
+
+```sh
```

Comment:
```suggestion
```bash
```

Reply:
see above, they can't copy and paste this one because it requires input for command